public/js/selfoss-events-navigation.js

```diff
@@ -71,7 +71,7 @@ selfoss.events.navigation = function() {
         $(this).addClass('active');
         
         if($(this).hasClass('nav-tags-all')==false) {
-            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html();
+            location.hash = '#' + selfoss.filter.type + '/tag-' + $(this).find('span').html().replace('%', '%25');
```

Comment:
This should probably be `encodeURIComponent($(this).find('span').html())`. Works both in Chromium and Firefox.